packages/release/logdog.common.conf

```diff
@@ -17,4 +17,4 @@ exec signpost signpost status
 exec wicked wicked show all
 file os-release /etc/os-release
 glob /var/log/kdump/*
-settings settings.json
+settings settings.json
```

Comment:
Can we move the `exec wicked` line to the wicked package? Then we could add a separate `logdog.conf` drop-in to the networkd package and fix #3514.